Overview.src.html

```diff
@@ -2031,6 +2031,17 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
  <span data-anolis-ref>MIX</span>
  <span data-anolis-ref>CSP</span>
 
+ <li><p>If <var>request</var>'s <span title=concept-request-referrer-policy>referrer policy</span>
+ is the empty string and <var>request</var>'s
+ <span title=concept-request-client>client</span> is non-null, set <var>request</var>'s
+ <span title=concept-request-referrer-policy>referrer policy</span> to <var>request</var>'s
+ <span title=concept-request-client>client</span>'s associated referrer policy.
```

Comment:
Where is "associated referrer policy" defined? Would be nice to directly reference it.